api_tests/lib/ethereum.ts

```diff
@@ -81,31 +86,38 @@ export class EthereumWallet {
         return this._address;
     }
 
-    async fund(ethAmount: BN) {
+    async fund(ethAmount: string) {
         const parity_dev_account = "0x00a329c0648769a73afac7f9381e08fb43dbea72";
         const parity_dev_password = "";
-        const weiAmount = _web3Client.utils.toWei(ethAmount, "ether");
+
+        const weiAmount = utils.toWei(ethAmount, "ether");
```

Comment:
:wrench: If you would use `let` you could overwrite the variable.

Reply:
They have different meaning as one is in wei and other in Ether so happy not to override.